### ISSUE TITLE
Aura not applying to self

### ISSUE THREAD

**Issue by cbgh2021** [kandashi/Active-Auras#232]
The aura is not applying to the token it's attached to, nor is is applying to a duplicate of that token. Does apply to other tokens.

**To Reproduce**
1. Set aura to 5ft
2. target friendly
3. did not check "ignore self?" box

**Expected behavior**
should apply the aura to the token and all similar tokens. (this case a skeleton defender that applies AC bonus to self and nearby allies) It does affect tokens of a different type, but does not affect multiples of the same defender.

**Environment:**
   - Foundry Version 269
 - Module Version 4.13
 - All Modules Disabled: Yes/No
 - Module List if no:

**Comment by KellethDregar** [comment]
I'm also encountering this issue but for me I've noticed it transfers things like Aura of Courage's frightened immunity, and it transfers adding a solid number but not with a variable like +@abilities.cha.mod

**Comment by tprussak** [comment]
This may actually just be an issue with DAE, even without an aura trying to add +@abilities.cha.mod to saves does not work.

**Comment by tprussak** [comment]
Yes, this is an issue with DAE. It likely will roll correctly, but not display in the sheet, if using the CUSTOM mode, see https://gitlab.com/tposney/dae/-/issues/327. You can also do +[[@abilities.cha.mod]] to force it to pre-evaluate.

**Comment by Gib2024** [comment]
It is not adding to the display of the saves but is working when you make a save;
![image](https://github.com/user-attachments/assets/15ceb106-9317-4c64-935f-f8f99efd7d0c)
![image](https://github.com/user-attachments/assets/a47d641b-1feb-42be-83d9-973f65bd81da)

**Comment by MrPrimate** [comment]
> I don't have DAE since it is not updated for D&D v4 yet.  It is not adding to the display of the saves but is working when you make a save;
> ![image](https://github.com/user-attachments/assets/15ceb106-9317-4c64-935f-f8f99efd7d0c)
> ![image](https://github.com/user-attachments/assets/a47d641b-1feb-42be-83d9-973f65bd81da)
> 

I believe this is a big with the character sheet, not active auras, and will happen for a normal bonus applied to that actors saves with AA disabled
